src/sentry/incidents/models.py

```diff
@@ -848,6 +882,19 @@ class Meta:
         db_table = "sentry_alertruleactivity"
 
 
+@register_alert_subscription_callback(AlertRuleMonitorType.ACTIVATED)
```

Comment:
if we want to keep the key as a `int`, we can do `AlertRuleMonitorType.ACTIVATED.value`